conf/messages.cy

```diff
@@ -69,3 +69,17 @@ notOnPrivateBeta.notSignedUp.p1.preLink = You can choose to
 notOnPrivateBeta.notSignedUp.p1.link = take part in the EMCS private beta research
 notOnPrivateBeta.notSignedUp.p1.afterLink = if you are not already involved.
 notOnPrivateBeta.notSignedUp.p2 = To take part you must currently submit EMCS receipts using the HMRC platform.
+
+arc.subHeading = Explanation for delay to {0}
```

Comment:
Copydeck says "Explain a delay to [ARC]" but prototype says "Explanation for delay to [ARC]" - worth checking which is correct with UCD guys.

Reply:
Asked UCD, see reply https://hmrcdigital.slack.com/archives/C03NB3T2D42/p1688734280541539?thread_ts=1688734226.950419&cid=C03NB3T2D42